services-custom/s3-transfer-manager/src/main/java/software/amazon/awssdk/transfer/s3/UploadDirectoryOverrideConfiguration.java

```diff
@@ -66,14 +64,6 @@ public Optional<Integer> maxDepth() {
         return Optional.ofNullable(maxDepth);
     }
 
-    /**
-     * @return whether to recursively upload all files under the specified directory
```

Comment:
Query : for `public Optional<Integer> maxDepth()` java doc should we mention how this can be used to disable disable recursive uploads of directory ?

Reply:
I think this is covered on the setter already:
```
         * Specify the maximum number of levels of directories to visit. Must be positive.
         * 1 means only the files directly within the provided source directory are visited.
```